openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -23,13 +23,7 @@
     <header class="page-header has-secondary">
         <div class="page-header-main">
             <nav aria-label="${_('Course Outline')}" class="sr-is-focusable" tabindex="-1">
-                % if waffle.flag_is_active(request, UNIFIED_COURSE_EXPERIENCE_FLAG):
-                    <h2 class="hd hd-3 page-title">
-                        ${_("{course_name}").format(org=course.display_org_with_default, course_name=course.display_name_with_default)}
```

Comment:
Thanks for the clean-up.  The old code was confusing, and I won't even explain what I thought it might be doing.